src/NuGetTrends.Data/NuGetTrends.Data.csproj

```diff
@@ -11,8 +11,6 @@
 
   <ItemGroup>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.4" />
```

Comment:
I guess we should also upgrade to `3.0.1` here? There are some problems that could happen with queries being evaluated in-memory now throwing exceptions.. but I don't think we do any complicated query so we should be fine.

Reply:
good point. Let's do that after some tests